content/en/cosign/registry_support.md

```diff
@@ -22,7 +22,7 @@ Today, Cosign has been tested and works against the following registries:
 * Sonatype Nexus Container Registry
 * Alibaba Cloud Container Registry
 
-We aim for wide registry support. To sign images in registries which do not yet fully support [OCI media types](https://github.com/sigstore/cosign/blob/main/SPEC.md#object-types), one may need to use `COSIGN_DOCKER_MEDIA_TYPES` to fall back to legacy equivalents. For example:
+We aim for wide registry support. To sign images in registries which do not yet fully support OCI media types, one may need to use `COSIGN_DOCKER_MEDIA_TYPES` to fall back to legacy equivalents. For example:
```

Comment:
This link and SPEC.md no longer exist (https://github.com/sigstore/cosign/blob/main/SPEC.md#object-types)